product_docs/docs/eprs/7/eprs_rel_notes/16_eprs_rel_notes_7.2.1.mdx

```diff
@@ -0,0 +1,11 @@
+---
+title: "Version 7.2.1"
+---
+
+
+New features, enhancements, bug fixes, and other changes in Replication Server 7.2.1 include the following:
+
+| Type    | Description |
+| ------- |------------ |
+| Bug fix | Fixed the issue that caused the Publication creation failure due to a shadow table name conflict. #83414 |
```

Comment:
It's my understanding that the support case number should be listed after the sentence using this convention:
"[Support Ticket #<case number]"

Please update this note to use this convention.